entrypoint.sh

```diff
@@ -37,24 +36,40 @@ fi
 
 # Deploy the Fly app, creating it first if needed.
 if ! flyctl status --app "$app"; then
+  # Backup the original config file since 'flyctl launch' messes up the [build.args] section
+  cp "$config" "$config.bak"
   flyctl launch --no-deploy --copy-config --name "$app" --image "$image" --region "$region" --org "$org"
-  if [ -n "$INPUT_SECRETS" ]; then
-    echo $INPUT_SECRETS | tr " " "\n" | flyctl secrets import --app "$app"
-  fi
-  flyctl deploy --app "$app" --region "$region" --image "$image" --region "$region" --strategy immediate
-elif [ "$INPUT_UPDATE" != "false" ]; then
-  flyctl deploy --config "$config" --app "$app" --region "$region" --image "$image" --region "$region" --strategy immediate
+  # Restore the original config file
+  cp "$config.bak" "$config"
 fi
+if [ -n "$INPUT_SECRETS" ]; then
+  echo $INPUT_SECRETS | tr " " "\n" | flyctl secrets import --app "$app"
+fi
+
+echo "Contents of config $config file: " && cat "$config"
+flyctl deploy --config "$config" --app "$app" --region "$region" --image "$image" --strategy immediate
```

Comment:
The problem I'm looking at right now is that for a first time PR deployment, the deploy step needs to come after the `flyctl postgres attach` because the DATABASE_URL will not exist yet causing the deploy to fail.

Reply:
Updated.